deb/openmediavault/var/www/openmediavault/js/omv/module/admin/storage/mdadm/Mdadm.js

```diff
@@ -199,7 +200,11 @@ Ext.define("OMV.module.admin.storage.mdadm.device.Create", {
 					stateId: "serialnumber",
 					flex: 1
 				}]
-			}
+			},
+			plugins: [{
+				ptype: "fieldinfo",
+				text: _("Select the devices that will be used to create the RAID device. Devices connected via USB will not be listed.")
```

Comment:
Can we replace `Devices connected via USB` with `Devices connected via USB (too unreliable)`? Otherwise fine with me.